plugins/xdebug/xdebug.cc

```diff
@@ -409,21 +413,73 @@ XInjectResponseHeaders(TSCont /* contp */, TSEvent event, void *edata)
   return TS_EVENT_NONE;
 }
 
+static bool
+isFwdFieldValue(const char *value, int valueLen, intmax_t &fwdCnt)
+{
+  ts::TextView paramName("fwd");
+
+  if (valueLen < static_cast<int>(paramName.size())) {
```

Comment:
If you initialized `val` before this, you wouldn't have to cast.

Reply:
What I should construct **val** before I actually need it, just to avoid a cast?  The cast would not be so bad if I could just use a constructor style cast, but we can't because of the Leif rule.  I just embrace the cast-misery.